rasa/core/actions/action.py

```diff
@@ -166,6 +166,7 @@ async def run(
         nlg: "NaturalLanguageGenerator",
         tracker: "DialogueStateTracker",
         domain: "Domain",
+        metadata: Optional[Dict] = None,
```

Comment:
this one and all following changes in run() are not neccessary now.

Reply:
Done, was the hurry... sorry.